code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -227,6 +227,8 @@
 		health -= W.force
 		add_fingerprint(user)
 		healthcheck()
+		playsound(loc, 'sound/weapons/smash.ogg', 60, 1)
+		..()
 	else
 		..()
```

Comment:
You can clean this up by removing the else and just calling ..() after the if statement runs.